native/src/utils/object.js

```diff
@@ -0,0 +1,11 @@
+// @flow
+
+export const values = <T> (object: $ReadOnly<{ [string]: T }>): Array<T> => {
+  // $FlowFixMe
+  return Object.values(object)
```

Comment:
Would also be possible to use the following in order not to have $FlowFixMes, similar for entries below:
```
  return Object.keys(object).map(key => object[key])
```